.github/workflows/actions.yml

```diff
@@ -23,11 +23,47 @@ jobs:
         with:
           profile: minimal
       - name: Build
-        run: cargo build --verbose --all-features
+        run: cargo build --release --verbose --all-features
```

Comment:
This means we are building twice for CI to pass, could we make the build step not affect CI check status maybe?

Reply:
Right, good point.  So we can either:

1. Run tests in release mode (`cargo test --release`) or
2. Run tests as is, in debug mode and only do `cargo build --release` when pushing to the `master` branch.

Which one do you prefer?  I don't see anything wrong with 1. personally and it's going to be quicker to execute overall.

Reply:
I would prefer (2) because we don't have any tests currently, so (1) would just make ci slower.